src/resources/dialogs/dialogs.scss

```diff
@@ -83,6 +82,9 @@ body > ux-dialog-container {
     }
 
     &.disclaimer {
+      .marginRight10 {
+        margin-inline-end: 10px;
```

Comment:
Lets just use margin-right here.  We only support english, don't know and aren't testing (and don't want to test) the implications of using margin-inline-end on this element for all cases, and don't want to introduce this kind of thing into the app without a clear plan/intention and being consistent with it across the app.